pages/_app.js

```diff
@@ -18,6 +19,9 @@ if (typeof window !== 'undefined') {
 export default function MyApp({ Component, pageProps, currentVersion, versions }) {
   useJssStyles()
 
+  // We add mixpanel id to every moovweb.app anchor
+  AddMixpanelIds()
```

Comment:
Please rename this useMixpanel() so it looks more like a hook (which is really what it is based on the way it's written.)